entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityAttributeMapping.java

```diff
@@ -30,14 +36,23 @@
         config.getConfigList(ATTRIBUTE_MAP_CONFIG_PATH).stream()
             .collect(
                 toUnmodifiableMap(
-                    conf -> conf.getString("name"), conf -> conf.getString("subDocPath"))));
+                    conf -> conf.getString("name"), conf -> conf.getString(SUB_DOC_PATH))),
+        config.getConfigList(ATTRIBUTE_MAP_CONFIG_PATH).stream()
+            .filter(
+                conf ->
+                    conf.hasPath(VALUE_KIND)
```

Comment:
These can't be stored in a flat list - should be in a map keyed by scope since there may be duplicate keys with different types in different scopes.